wire-ios/Wire-iOS/Sources/Managers/ColorSchemeController.swift

```diff
@@ -19,22 +19,13 @@
 import Foundation
 import WireSyncEngine
 
-extension Notification.Name {
-    static let colorSchemeControllerDidApplyColorSchemeChange = Self("ColorSchemeControllerDidApplyColorSchemeChange")
-}
-
 final class ColorSchemeController: NSObject {
 
     var userObserverToken: Any?
 
     init(userSession: UserSession) {
         super.init()
 
-        // When SelfUser.provider is nil, e.g. running tests, do not set up UserChangeInfo observer
```

Comment:
question: what does ColorSchemeController do? and why is it not observing accentColor?